.gitignore

```diff
@@ -0,0 +1,6 @@
+kubeconfig
```

Comment:
admin.conf? maybe

Reply:
Keeping this consistent with the implementation in kubeaddons-enterprise for now.